### ISSUE TITLE
Nonpolar solvation energy in OpenMM Implicit Solvent models

### ISSUE THREAD

**Issue by membiophys** [openmm/openmm#4053]
Hi !
I have a question about the nonpolar contribution (usually proportional to SASA) in the implicit solvent models implemented in OpenMM. I cannot find much information about it in documentation or tutorials, apart from the small description of parameter `gbsaModel()` in the doc page about the _class_ `simtk.openmm.app.amberprmtopfile.AmberPrmtopFile(file)`. But there, it just mentions the option `gbsaModel (str='ACE')`  as the only possible model of nonpolar solvation, with no explanation about the characteristics of that model. Where can I find information about the 'ACE' model? Are there additional parameters to control this model, to implicitly account for differences in solvent properties that affect the nonpolar contribution. For example, in Amber you can scale the nonpolar contribution via the `surften` parameter.

**Comment by peastman** [comment]
See

http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html#gbsaobcforce

The ACE approximation was introduced in equation 2 of

https://www.sciencedirect.com/science/article/abs/pii/S0022283698921726

though the formula in OpenMM is slightly different: see #4016.

Depending on the GB model you select, it might be implemented with either a GBSAOBCForce or a CustomGBForce.  The former lets you set the scale factor by calling `setSurfaceAreaEnergy()`.  With the former, it's contained within the energy expression.

https://github.com/openmm/openmm/blob/86f0708200c1e700d241957d2280b0344a134e8a/wrappers/python/openmm/app/internal/customgbforces.py#L374-L375

The value 28.3919551 is $4\pi\sigma$.

**Comment by membiophys** [comment]
Oh, I see. Thank you, and sorry I missed #4016. I have it clear now.

Funny enough, there is a mistake in web/html version of [Eq. 2 from 1998 Shaefer's paper](https://doi.org/10.1006/jmbi.1998.2172), where the last term shows as $B_i  ~b_i$, instead of $\frac{R_1}{b_i}$. But it is correct in the pdf version of the paper.

**Comment by peastman** [comment]
It might be good to notify the journal and see if they'll fix it.

Is it ok to close this issue?

**Comment by membiophys** [comment]
Yes, I'll write to the Journal to avoid confusion to other people.
Please, fill free to close the issue. Thank you again for your quick and valuable help!

**Comment by peastman** [comment]
Glad to help!

**Comment by pitmanme** [comment]
> See
> 
> http://docs.openmm.org/latest/userguide/theory/02_standard_forces.html#gbsaobcforce
> 
> The ACE approximation was introduced in equation 2 of
> 
> https://www.sciencedirect.com/science/article/abs/pii/S0022283698921726
> 
> though the formula in OpenMM is slightly different: see #4016.
> 
> Depending on the GB model you select, it might be implemented with either a GBSAOBCForce or a CustomGBForce. The former lets you set the scale factor by calling `setSurfaceAreaEnergy()`. With the former, it's contained within the energy expression.
> 
> https://github.com/openmm/openmm/blob/86f0708200c1e700d241957d2280b0344a134e8a/wrappers/python/openmm/app/internal/customgbforces.py#L374-L375
> 
> The value 28.3919551 is 4πσ.

@peastman Do I interpret this correctly that this term would be constant during simulation?

**Comment by peastman** [comment]
No, it depends on the particle positions.  Otherwise there would be no need to include it.
